tests/infra/repos/postgresql/guardian-account.spec.ts

```diff
@@ -0,0 +1,43 @@
+
+import { GuardianAccountRepository } from '../../../../src/infra/repos/postgresql/guardian-account-repository'
```

Comment:
Remove "../../../src/" from import, because the `src` folder is already set as the root of the project in typescript